storage/migrations/22_consensus_accounts_first_activity.up.sql

```diff
@@ -0,0 +1,6 @@
+BEGIN;
```

Comment:
I swapped accounts_list migration and this new one, so that this one comes first. That way we can use `first_activity` in the account_lists without writing one more migration.

This should be fine since account-lists migration was not yet released/deployed anywhere.

Reply:
has accounts_list never been deployed?

Reply:
I believe so yes, it has been merged only last week.

Reply:
ohh if it's never been in a released version that's even better